.changeset/little-berries-explode.md

```diff
@@ -0,0 +1,44 @@
+---
+'@iroha2/data-model': major
+---
+
+**refactor!**: join all data-model types as `datamodel` namespace export
+
+##### What is the change
+
+If previously all data-model types were imported directly from the package:
+
+```ts
+import { AccountId, DomainId } from '@iroha2/data-model'
+
+const acc = AccountId({
+  name: 'alice',
+  domain_id: DomainId({
+    name: 'wonderland',
+  }),
+})
+```
+
+Now all data-model types are joined into a single `datamodel` namespace:
+
+```ts
+import { datamodel } from '@iroha2/data-model'
+
+const acc = datamodel.AccountId({
+  name: 'alice',
+  domain_id: datamodel.DomainId({
+    name: 'wonderland',
+  }),
+})
+```
+
+##### Why the change was made
+
+For a couple of reasons:
```

Comment:
I don't remember how it's done in other sections like this but I feel like you can remove this part (the header is enough)
```suggestion
```